src/js/rs-release/rs-release.js

```diff
@@ -0,0 +1,130 @@
+    },
+
+    createContent: function (releaseVersion) {
+        var releaseData;
+        EvaManager.get({
+            host:EVA_RELEASE_HOST,
+            version: EVA_VERSION,
+            category: 'stats',
+            resource: 'per-species',
+            params: {releaseVersion: releaseVersion},
+            async: false,
+            success: function (response) {
+                try {
+                    releaseData = response;
+                } catch (e) {
+                    console.log(e);
+                }
+            }
+        });
+
+        var releaseInfo;
+        EvaManager.get({
+            host:EVA_RELEASE_HOST,
+            version: EVA_VERSION,
+            category: 'info',
+            params: {releaseVersion: releaseVersion},
+            async: false,
+            success: function (response) {
+                try {
+                    releaseInfo = response[0];
+                } catch (e) {
+                    console.log(e);
+                }
+            }
+        });
+
+        var table = '<div><h2>Clustered variants (RS) Release</h2></div>' +
+                    '<div class="callout success">' +
+                        '<p>' +
+                            'The RS ID release (v.' + releaseVersion + ') is now available in our FTP. ' +
+                            '<a href="' + releaseInfo.releaseFtp +'" target="_blank">[View release]</a>' +
+                        '</p>' +
+                    '</div>' +
+                    '<p>' + releaseInfo.releaseDescription + '</p>' +
+                    '<table id="rs-release-table" class="responsive-table hover tablesorter table-fixed">' +
+                        '<thead>' +
+                            '<tr>' +
+                                '<th>Scientific name</th>' +
+                                '<th>Taxonomy ID</th>' +
+                                '<th><div title="RS IDs that can be browsed on the EVA websites">Current RS</div></th>' +
+                                '<th><div title="RS IDs that are resolved to multiple location on the genome">Multi-mapped RS</div></th>' +
+                                '<th><div title="RS IDs that should NOT be used because they are merged into another active RS">Merged RS</div></th>' +
+                                '<th><div title="RS IDs that should NOT be used since these RS IDs were deprecated">Deprecated RS</div></th>' +
+                                '<th><div title="RS IDs that should NOT be used because they have been merged into a deprecated RS">Merged Deprecated RS</div></th>' +
+                                '<th><div title="RS IDs that could NOT be mapped against an assembly">Unmapped RS</div></th>' +
+                            '</tr>' +
+                        '</thead>' +
+                        '<tbody>';
+
+        _.each(releaseData, function (species) {
+            var scientificNameNormalized = species.scientificName.toLowerCase().replace(' ', '_');
```

Comment:
I suggest to replicate the functionality [here](https://github.com/EBIvariation/eva-accession/pull/246/files/23e335472055f8bfe29fa714d12250b0c9cae030#diff-b6d7f7cf30896264d9e49bd1ebe1bed7e80706b5ecf202be868fad3ad451a245R110) as inelegant as it is :(

Reply:
The release folder was included in the database so the API is now retrieving that and constructing the links. The website is not constructing the links anymore.